Makefile

```diff
@@ -85,9 +91,12 @@ acceptance: ## Run all acceptance tests
 	@cd "$${ACCEPTANCE_WORKDIR}"
 	@go run acceptance/coverage/coverage.go .
 	@$(MAKE) build
+ifneq ($(BUILD_IMG_ARCH),$(BUILD_LOCAL_ARCH))
+	@$(MAKE) build-for-test
```

Comment:
Not sure we need this, test will target `push-image` -> `push_image_$(BUILD_IMG_ARCH)` -> `image_$(BUILD_IMG_ARCH)` -> `$(ALL_SUPPORTED_OS_ARCH)`, or when that is expanded: `push-image` -> `push_image_linux_amd64` -> `image_linux_amd64`. I think...

I think the `kindCluster.buildCliImage` will overwrite the image anyhow. I think...